src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,6 @@
+$color-card-blue: #34568b;
+$color-grey: #ccc;
+$color-nav: #060b35;
+$color-nav-hover: #00acdc;
+$transform-delay: transform 0.3s;
+$color-delay: color 0.3s;
```

Comment:
It's not necessary to create variables for transform and color. It would be better to create a variable for delay (0.3s) and use it